data/competencies.yml

```diff
@@ -446,6 +455,7 @@
     - When pressed for time, focuses on ensuring test coverage of the most critical system functionality
     - Manages technical debt, understands consequences of technical debt vs the cost of fixing it and acts accordingly
     - Can explain when something is worth refactoring even when it will impact the speed of delivery
+    - knows when to use an IM user or a role in AWS
```

Comment:
After reviewing this, we think maybe all the examples in this competency aren't right. All of the existing examples talk about technical vs delivery trade-offs rather than purely technical. I think this should be included in another issue, so maybe remove from this one in the interest of merging it.

```suggestion
```